test/gpu/ffmpeg_test.go

```diff
@@ -50,26 +40,16 @@ func TestFffmpegEncodeGPU(t *testing.T) {
 // TestFffmpegDecodeGPU runs ffmpeg in a GPU container using NVDEC.
 func TestFffmpegDecodeGPU(t *testing.T) {
 	ctx := context.Background()
-	isGVisor, err := dockerutil.IsGVisorRuntime(ctx, t)
-	if err != nil {
-		t.Fatalf("Failed to determine if runtime is gVisor: %v", err)
-	}
-	if isGVisor {
-		t.Skip("This test is currently broken in gVisor")
-	}
 	container := dockerutil.MakeContainer(ctx, t)
 	defer container.CleanUp(ctx)
-	opts, err := dockerutil.GPURunOpts(dockerutil.SniffGPUOpts{
-		Capabilities:           "NVIDIA_DRIVER_CAPABILITIES=video",
-		AllowIncompatibleIoctl: true, // TODO(gvisor.dev/issue/9452): Remove once supported in gVisor.
-	})
+	opts, err := dockerutil.GPURunOpts(dockerutil.SniffGPUOpts{Capabilities: "NVIDIA_DRIVER_CAPABILITIES=compute,video"})
 	if err != nil {
 		t.Fatalf("Failed to get GPU run options: %v", err)
 	}
 	opts.Image = "benchmarks/ffmpeg"
 	// h264_cuvid refers to NVDEC. See Section 4.2 in
 	// https://docs.nvidia.com/video-technologies/video-codec-sdk/pdf/Using_FFmpeg_with_NVIDIA_GPU_Hardware_Acceleration.pdf
-	cmd := strings.Split("ffmpeg -y -vsync 0 -c:v h264_cuvid -i video.mp4 output.yuv", " ")
+	cmd := []string{"bash", "-c", "ffmpeg -i video.mp4 -c:v h264_nvenc -preset fast output.mp4 && ffmpeg -y -vsync 0 -c:v h264_cuvid -i output.mp4 video.mp4"}
```

Comment:
Encoding and decoding should be two separate unit tests. (They can use the same image)